### PR TITLE
feat: Introduced VoiceCardContent component and a few new icons to su…

### PR DESCRIPTION
**Pull request by mnasyrov**
…pport Audio Transcription feature.

Signed-off-by: Mikhail Nasyrov <m.nasyrov@dlg.im>

### CODE REVIEW

File: src/components/MessageContent/Voice/Voice.css [hunk 1]
```diff
@@ -19,17 +18,29 @@
 }
 
 /* Styles */
-.container {
+.card {
```
Comment (meafmira) [inline src/components/MessageContent/Voice/Voice.css]: Use `voidCard` prefix for your classes to pass "grep test"

File: src/components/Icon/svg/speech_recognition.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (olegshilov) [inline src/components/Icon/svg/speech_recognition.svg]: Optimize image.

File: src/components/MessageContent/Voice/Voice.css (hunk 1 above)
Comment (mnasyrov) [reply src/components/MessageContent/Voice/Voice.css]: done.

File: src/components/Icon/svg/speech_recognition.svg (hunk 2 above)
Comment (mnasyrov) [reply src/components/Icon/svg/speech_recognition.svg]: Removed unnecessary SVG metadata.

**Comment by meafmira** [comment]
@mnasyrov verify your commits pls

**Comment by mnasyrov** [comment]
> @mnasyrov verify your commits pls

Done.

**Pull request merged**